pt/santander.py

```diff
@@ -0,0 +1,362 @@
+            d["contact:youtube"] = nd["socialData"].get("youtubeLink", "")
+            d["contact:facebook"] = nd["socialData"].get("facebookLink", "")
+            d["contact:twitter"] = nd["socialData"].get("twitterLink", "")
+            d["contact:linkedin"] = nd["socialData"].get("linkedinLink", "")
+            d["contact:instagram"] = nd["socialData"].get("instagramLink", "")
+            d["contact:tiktok"] = nd["socialData"].get("tiktokLink", "")
+            assert nd["socialData"].get("googleLink", "") == ""
+            
+        # Schedule
+        opening_hours = ""
+        schedule = nd["schedule"]
+        workingDays = {}
+        weekday2str = {
+            "MONDAY": "Mo",
+            "TUESDAY": "Tu",
+            "WEDNESDAY": "We",
+            "THURSDAY": "Th",
+            "FRIDAY": "Fr",
+            "SATURDAY": "Sa",
+            "SUNDAY": "Su"
+        }
+        for workday in schedule["workingDay"]:
+            wd = weekday2str[workday]
+            if schedule["workingDay"][workday] == []:
+                continue
+            workingHours = ",".join(fixWorkingHours(schedule["workingDay"][workday]))
+            if workingHours == "Encerrado":
+                continue
+            if workingHours not in workingDays:
+                workingDays[workingHours] = []
+            workingDays[workingHours].append(wd)
+        for workingHours, days in workingDays.items():
+            days.sort(key=lambda s: DAYS.index(s))
+            if opening_hours != "":
+                opening_hours += "; "
+            if days == ["Mo", "Tu", "We", "Th", "Fr"]:
+                days = ["Mo-Fr"]
+            opening_hours += ",".join(days) + " " + workingHours
+        days_off = []
+        for day in schedule["specialDay"]:
+            if day["time"] == [None]:
+                month, day = day["date"].split("-")
+                monthName = {
+                    "01": "Jan",
+                    "02": "Feb",
+                    "03": "Mar",
+                    "04": "Apr",
+                    "05": "May",
+                    "06": "Jun",
+                    "07": "Jul",
+                    "08": "Aug",
+                    "09": "Sep",
+                    "10": "Oct",
+                    "11": "Nov",
+                    "12": "Dec",
+                    "13": "Dec",
+                }[month]
+                days_off.append(f"{monthName} {day}")
+            else:
+                raise Exception("Special days not implemented")
+        if days_off != []:
+            opening_hours += "; " + ",".join(days_off) + " off"
+        d["opening_hours"] = opening_hours
+
+        if "urlDetailPage" in nd:
+            d["website"] = nd["urlDetailPage"]
+            
+        if nd["poicode"] not in d[REF].split(";"):
+            d["ref"] = nd["poicode"]
+
+        # Services
+        if "comercialProducts" in nd:
+            commercialProducts = set([product["default"] for product in nd["comercialProducts"]])
```

Comment:
Force the case to reduce the number of conditions below, like e.g. separate branches for "Levantamentos" and "LEVANTAMENTOS":
```suggestion
            commercialProducts = {product["default"].upper() for product in nd["comercialProducts"]}
```